### PR TITLE
# CP-5 fix the eslint bug

### PR DESCRIPTION
**Pull request by samyshyaka**
#### What the PR does
Fix the Eslint bug

#### Description of Tasks to be completed

- Update the Eslint rules
- Restore the affected code in the src folder

#### How this should be manually tested
Clone the project repository linked [here](https://github.com/atlp-rwanda/phantom-fe-codebandits)

```
cd phantom-fe-codebandits/
git branch bg-eslint-bug-fix-cp-5
git checkout bg-eslint-bug-fix-cp-5
git pull origin bg-eslint-bug-fix-cp-5
npm install
git add 'filename'  
git commit -m "commit message"
```

#### Relevant Trello Card
[#CP-5](https://trello.com/c/etXFB94I/5-fix-the-eslint-bug)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,9 +11,6 @@
     "format": "npx pretty-quick",
     "lint": "npx eslint src/*.js --fix"
```
Comment (feyton) [inline package.json]: ## Support for **JSX** files.
Add a Regex selector to check for linting issues in both js and jsx.

Edit the script to match this:
"lint": "npx eslint src/*.{js,jsx} --fix"
@samuel-shyaka-dus

File: styleguide.md [hunk 2]
```diff
@@ -18,4 +18,5 @@
 - Body: 
     Family: [Open Sans](https://fonts.google.com/specimen/Open+Sans)
     Size: 16px
-    Weight: 400 
\ No newline at end of file
+    Weight: 40
```
Comment (feyton) [inline styleguide.md]: Fix the typo font-weight. The value should be in the range of **100-900** at least

**Review by feyton — CHANGES_REQUESTED** [review]
If in the future we plan to combine the JS and JSX, then adding the aforementioned script would be great.

**Review by feyton — APPROVED** [review]
The script and typos are fixed

**Review by Yvad60 — APPROVED** [review]
working now

**Comment by samyshyaka** [comment]
> The script and typos are fixed

Great

**Pull request merged**